File: RAI/InnerClass/JAVA/Exterior.java
public class Exterior{ //Outer Class

    public String mensaje;

    class Coordenada{
        public int x;
        public int y;

        public Coordenada(int x, int y){
            this.x=x;
            this.y=y;
        }
    }

    public Exterior(String mensaje){
        this.mensaje = mensaje;
    }

}

File: RAI/InnerClass/python/Exterior.py
class Exterior:
    def __init__(self, mensaje):
        self.mensaje = mensaje
        #self.creador = self.Coordenada(x,y)
    
    def mostrar(self):
        print("Este es el mensaje de la clase exterior: " + self.mensaje)

    class Coordenada:
        def __init__(self, x, y):
            self.x = x
            self.y = y
        
        def mostrar(self):
            print("Coordenada [" + str(self.x) + "," + str(self.y) + "]")